lunes_cms/cms/widgets.py

```diff
@@ -3,24 +3,30 @@
 
 
 class ManyToManyOverlay(FilteredSelectMultiple):
-    """Adds an overlay to the `FilteredSelectMultiple`
+    """
+    Adds an overlay to the `FilteredSelectMultiple`
     widget to preview images and audios of Document objects.
     Inherits from `django.contrib.admin.widgets.FilteredSelectMultiple`.
     """
 
     class Media:
+        """
+        Media class of ManyToManyOverlay
+        """
+
         js = (
             "js/manytomany_overlay.js",
             "js/overlay.js",
             reverse_lazy("javascript-translations"),
         )
         css = {"all": ("css/overlay.css",)}
 
+    # pylint: disable=return-in-init
```

Comment:
```suggestion
```
In this case I think we can safely remove the `return` since the parent `__init__()` method does also not return anything.